### ISSUE TITLE
GraphDB hits into query timeouts during load-tests 

### ISSUE THREAD

**Issue by subotic** [dhlab-basel/Knora#937]
The query timeout is currently set at 5 seconds:

```
owlim:query-timeout "5" ;
```

This will need some tweaking.

**Comment by tobiasschweizer** [comment]
The thing is that it should protect GraphDB from malicious Gravsearch queries, but the limit should not be reached when doing reasonable things.

"Quadratur des Kreises"

**Comment by benjamingeer** [comment]
> Maybe we could dump the calls to GraphDB done during the load-test into files and then simply run those?

Definitely. Then it would be easier to send the test case to Ontotext to ask them about it.

**Comment by subotic** [comment]
We are not the first ones to have this use case: https://github.com/buger/goreplay

Trying it out now to capture all requests to GraphDB. There are a lot of requests. After only a few minutes, the saved file is already around 150MB.
